src/notes/note-metadata.dto.ts

```diff
@@ -33,6 +34,12 @@ export class NoteMetadataDto {
   @ApiPropertyOptional()
   alias: string;
 
+  /**
+   * If the note is note from before version 2 was a thing
+   */
+  @IsBoolean()
+  preVersionTwoNote: boolean;
```

Comment:
Why do we have a boolean for this if we've got the actual version?
Wouldn't it make more sense to adjust the frontend implementation on this?

Reply:
That could also be done.